simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -714,18 +747,13 @@ public void after(TestInfo testInfo) throws IOException {
                 Files.move(Paths.get(pngPath), previousPng);
             }
 
-            XYChart totalRequestsPerServerPerNode =
-                    simulation.metricsReporter().chart(MetricNames.serverRequestMeterPattern());
-
-            XYChart totalRequestsPerClientPerEndpoint =
-                    simulation.metricsReporter().chart(MetricNames.perClientEndpointResponseTimerPattern());
-            SimulationMetricsReporter.png(
-                    pngPath,
-                    activeRequestsPerServerNode,
-                    totalRequestsPerServerPerNode,
-                    totalRequestsPerClientPerEndpoint
-                    // simulation.metrics().chart(Pattern.compile("(responseClose|globalResponses)"))
-                    );
+            charts.add(simulation.metricsReporter().chart(MetricNames.serverRequestMeterPattern()));
+
+            charts.addAll(simulation.metricsReporter().charts(MetricNames.perClientEndpointResponseTimerPattern()));
+
+            // charts.add(simulation.metrics().chart(Pattern.compile("(responseClose|globalResponses)")));
+
+            SimulationMetricsReporter.png(pngPath, charts);
```

Comment:
Let's break out any changes to the simulation output into a separate stacked/parallel PR, that way the simulation changes are more obviously coupled to improvements we make.

Reply:
Absolutely